services/frontend/src/components/admin/categoryTable/CategoryTableView.jsx

```diff
@@ -111,8 +111,9 @@ const CategoryTableView = ({
     filterIcon: (filtered) => (
       <SearchOutlined style={{ color: filtered ? "#1890ff" : undefined }} />
     ),
-    onFilter: (value, record) =>
-      record[dataIndex].toString().toLowerCase().includes(value.toLowerCase()),
+    onFilter: (value, record) => {
+      record[dataIndex].toString().toLowerCase().includes(value.toLowerCase());
+    },
```

Comment:
This function needs to return a boolean (https://ant.design/components/table/#Column) or else it won't filter the column